desk/app/groups-ui.hoon

```diff
@@ -285,7 +301,7 @@
   ==
 ++  get-suggested-contacts
   =+  .^(chat-running=? (scry %gu %chat /$))
-  =|  suggestions=(set ship)
+  =/  suggestions=(set ship)  manual-contact-suggestions
```

Comment:
This guy seems out of place in this PR. (^:

Reply:
Without this the PR generates merge conflicts. It is not great, I know, but in the end you still need to push those exact changes. I will be sure to push an explicit commit next time.